lib/core/sidebar/sidebar-action-menu.component.scss

```diff
@@ -56,8 +58,8 @@
         &-sidebar-action-menu-panel {
             margin-top: 7.5px;
             border-radius: 2px;
-            box-shadow: 0 8px 8px 0 rgba(0, 0, 0, 0.26), 0 0 8px 0 rgba(0, 0, 0, 0.12);
-            min-width: $adf-sidebar-action-menu-button-width !important;
+            box-shadow: 0 3px 1px -2px rgba(0,0,0,.2), 0 2px 2px 0 rgba(0,0,0,.14), 0 1px 5px 0 rgba(0,0,0,.12);
```

Comment:
this shadow will not works across all the theme can we get the shadow color from the theme?